### PR TITLE
Modify the convention of release name generation

### PR DESCRIPTION
**Pull request by shlomitk1**
Fixes https://github.com/fybrik/fybrik/issues/1998

### CODE REVIEW

File: manager/controllers/app/blueprint_controller.go [hunk 1]
```diff
@@ -326,7 +326,9 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, cfg *action.Configu
 		}
 
 		releaseName := managerUtils.GetReleaseName(managerUtils.GetApplicationNameFromLabels(blueprint.Labels),
-			managerUtils.GetApplicationNamespaceFromLabels(blueprint.Labels), instanceName)
+			managerUtils.GetApplicationNamespaceFromLabels(blueprint.Labels),
```
Comment (roytman) [inline manager/controllers/app/blueprint_controller.go:329]: why do we need the namespace? is uuid not enough
 ?

Comment (shlomitk1) [reply manager/controllers/app/blueprint_controller.go:329]: It probably is...

File: manager/controllers/app/blueprint_controller_unit_test.go [hunk 2]
```diff
@@ -87,9 +87,9 @@ func TestBlueprintReconcile(t *testing.T) {
 	g.Expect(cl.Get(context.TODO(), ns, blueprint)).To(gomega.BeNil(), "could not fetch the blueprint")
 	g.Expect(blueprint.Status.Releases).To(gomega.HaveLen(2))
 	g.Expect(blueprint.Status.Releases).
-		Should(gomega.HaveKeyWithValue("notebook-default-notebook-copy-batch", blueprint.Status.ObservedGeneration))
+		Should(gomega.HaveKeyWithValue("notebook1234-default-notebook-copy-batch", blueprint.Status.ObservedGeneration))
 	g.Expect(blueprint.Status.Releases).
```
[the remark this answers is not in the document]
Comment (shlomitk1) [reply manager/controllers/app/blueprint_controller_unit_test.go:91]: no, module names are different

File: manager/controllers/app/blueprint_controller_unit_test.go [hunk 3]
```diff
@@ -144,13 +145,15 @@ func TestLongReleaseName(t *testing.T) {
 	}
 
 	relName := utils.GetReleaseName(utils.GetApplicationNameFromLabels(blueprint.Labels),
-		utils.GetApplicationNamespaceFromLabels(blueprint.Labels), "ohandnottoforgettheflowstepnamethatincludesthetemplatenameandotherstuff")
+		utils.GetApplicationNamespaceFromLabels(blueprint.Labels), "",
+		"ohandnottoforgettheflowstepnamethatincludesthetemplatenameandotherstuff")
```
[the remark this answers is not in the document]
Comment (shlomitk1) [reply manager/controllers/app/blueprint_controller_unit_test.go:149]: ok

File: manager/controllers/app/blueprint_controller_unit_test.go [hunk 4]
```diff
@@ -144,13 +145,15 @@ func TestLongReleaseName(t *testing.T) {
...
 	g.Expect(relName).To(gomega.Equal("my-app-default-ohandnottoforgettheflowstepnamet-99207"))
```
[the remark this answers is not in the document]
Comment (shlomitk1) [reply manager/controllers/app/blueprint_controller_unit_test.go:150]: ok

File: manager/controllers/app/fybrikapplication_controller_unit_test.go [hunk 5]
```diff
@@ -198,7 +198,7 @@ func TestFybrikApplicationControllerCSVCopyAndRead(t *testing.T) {
 	connectionMap := endpoint.AdditionalProperties.Items
 	g.Expect(connectionMap).To(gomega.HaveKey("fybrik-arrow-flight"))
 	config := connectionMap["fybrik-arrow-flight"].(map[string]interface{})
-	g.Expect(config["hostname"]).To(gomega.Equal("read-path.notebook-default-arrow-flight-module.notebook"))
+	g.Expect(config["hostname"]).To(gomega.Equal("read-path.notebook1-default-arrow-flight-module.notebook"))
```
[the remark this answers is not in the document]
Comment (shlomitk1) [reply manager/controllers/app/fybrikapplication_controller_unit_test.go]: changed the tests/added explanations

**Review by roytman — APPROVED** [review]

**Review by roytman — APPROVED** [review]

**Pull request merged**